components/core/ApplicationUserControls.js

```diff
@@ -103,14 +103,21 @@ const STYLES_ITEM_BOX = css`
 `;
 
 const OpenIntercom = ({ user }) => {
```

Comment:
I made this fix myself b/c it was small, but just a heads up that you needed to destructure it here, otherwise it won't exist:

`const OpenIntercom = ({ user, onTogglePopup }) => {`